packages/wpcom-template-parts/src/utils.tsx

```diff
@@ -0,0 +1,42 @@
+import { translate as translateMethod } from 'i18n-calypso';
+
+export const getAutomatticBrandingNoun = ( translate: typeof translateMethod ) => {
+	const options = {
+		description:
+			'Branding to be shown on the Footer of the page, Automattic\'s variable will always contains the word "Automattic"',
```

Comment:
This `options.description` field, will it be correctly extracted by the `i18n` tool? I have some doubts about that. We might have to repeat it verbatim for every variant, however ugly that is.